contracts/PriceOracleMock.sol

```diff
@@ -75,40 +62,28 @@ contract PriceOracle is OpsManaged, PriceOracleInterface {
 
     /// @dev use this method to set price
     /// @param _price price
-    /// @return expirationHeight
+    /// @return uint256
     function setPrice(
```

Comment:
the function `setPrice` should just be removed for the PriceOracleMock; and likewise `setPrice` should be removed from `PriceOracleInterface` because the interface is only for the consumer (eg Pricer); and the consumer does not care about how the price gets set (as is clear now in the ConstantPriceOracleMock)

Reply:
Agreed, changes made.